packages/module/patternfly-docs/content/extensions/component-groups/examples/ContentHeader/ContentHeader.md

```diff
@@ -0,0 +1,20 @@
+---
+section: extensions
+subsection: Component groups
+id: Content header
+source: react
+propComponents: ['ContentHeader', 'ActionMenu']
+sourceLink: https://github.com/patternfly/react-component-groups/blob/main/packages/module/patternfly-docs/content/extensions/component-groups/examples/ContentHeader/ContentHeader.md
+---
+
+import ContentHeader from "@patternfly/react-component-groups/dist/dynamic/ContentHeader"
+
+The **content header** component should contain page title and description with a possibility to define additional badges, links, or icon separated with divider. Another option should be to display an actions menu and pass bredcrumbs.
+
+## Examples
+
+### Basic content header
+
+```js file="./ContentHeaderExample.tsx"
+
+```
```

Comment:
Let's add more examples - with an icon and without, how to define breadcrumbs, link and a badge...